html/rules.html

```diff
@@ -1,75 +1,75 @@
 <p><strong>Ignorance is not a excuse, read the rules.</strong></p>
-<p>0. The rules are the rules. Follow the spirit of the rules and use common sense, and you should have no problems.</p>
-<p>0.1 Know your faction lore and IC specific rules. If you play as any aligned faction, consider the server rules, superior to Lore.</p>
-<p>1. Don't be a dick. Play for fun, not to win.</p>
-<p>2. Do not use information gained outside of in character means. No Meta.</p>
-<p>3. Do not say in character (IC) things in the out of character (OOC) chat channel. Keep it IC.</p>
-<p>4. The role you spawn in as is the role you should play, joining as a role to desert or otherwise not play it, gets you a jobban.</p>
-<p>5. In-game administration rulings are final. If you have a problem with the ruling, take it to another member of staff. Fishing for admins is not recommended.</p>
-<p>6. Faction Metagaming. Do not move to the enemy factions base until at least 40 minutes into the round. Claiming to have &lsquo;found it&rsquo; is a bad excuse, and isn&rsquo;t going to work.</p>
-<p>7. Bad Deathclaw has a focus towards roleplay. Roleplay a believable character that could exist in the Fallout universe. Shitty memes and low roleplay gimmicks will not end well.</p>
-<p>8. Do not be unnecessarily hostile. Arguments are fine, being an asshole for no reason isn't. This includes taking it to OOC chat and making trouble.</p>
-<p>9. Do not attack or harm AFK/SSD players. You are free to take their items, but not attack them.</p>
-<p>10. No rule lawyering.</p>
+<p>0. Don’t be a Dick. If you are intentionally ruining another players experience or going out of your way to negatively impact the round, you can be punished for it..</p>
+<p>    
+	<br /><strong>General Rules</strong></p>
+<p>1. Admin Rulings are final. If you wish to contest a ruling, please go to the discord. In each situation, an admins ruling is how the rules are to be treated in that given situation.</p>
+<p>2. Don’t use OOC information IC. Don’t leak IC information to OOC.</p>
+<p>3. Play as the role you spawn in as, joining as a role to desert or otherwise undermine your faction is bannable.</p>
+<p>4. Bad Deathclaw is an RP server, and roleplay is expected. Failing to do so is punishable..</p>
+<p>4. Do not attack, assault or harm SSD/AFK players. You may take their equipment, but dragging them off or murdering them is not okay. Catatonic Players are free game.</p>
+<p>5. No Rules Lawyering.</p>

```

Comment:
This is both covered by the "admin's ruling is final" clause and doesn't explain what it actually means. Suggest removal.